### PR TITLE
fix(editorial): Add patch 2541786 for ds.

### PR DESCRIPTION
**Pull request by voidtek**
https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-330

### CODE REVIEW

File: resources/multisite_drupal_standard.make [hunk 1]
```diff
@@ -170,6 +170,10 @@ projects[diff][version] = "3.2"
 
 projects[ds][subdir] = "contrib"
 projects[ds][version] = "2.11"
+; Issue #2541786 : Changing layout to "fluid 2 columns stacked" resulted in crash
+; https://www.drupal.org/node/2541786
+; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-330
+projects[ds][patch][] = https://www.drupal.org/files/issues/2541786-8.patch
```
Comment (Fefaine) [inline resources/multisite_drupal_standard.make]: I disagree to solve this by patching 2.11
This patch is already included in 2.12
We should instead upgrade DS to 2.14
This is why It's important to include as much as possible module upgrades in 2.3 :)

**Comment by Fefaine** [comment]
:-1:

File: resources/multisite_drupal_standard.make (hunk 1 above)
Comment (voidtek) [inline resources/multisite_drupal_standard.make]: Ok.

**Comment by Fefaine** [comment]
To be replaced by DS module upgrade

**Pull request closed without merging**